tlcc/README.md

```diff
@@ -7,37 +7,22 @@ https://creativecommons.org/licenses/by/4.0/
 Before your continue here make sure you have built ``tlcc_enclave``.
 We refer to [tlcc_enclave/README.md](../tlcc_enclave).
 
-First we build tlcc as system chaincode plugin. What is that?
-See [here](https://hyperledger-fabric.readthedocs.io/en/release-1.4/systemchaincode.html).
-
-    $ make
-
-This builds `tlcc.so`. Using this plugin when running the peer inside
-Docker most probably will not work out-of-the-box, thus, not supported
-right now.
-
 ## Integrate with Fabric
 
-Add `tlcc` as system chaincode plugin to your `core.yaml`. Example:
-
-```
-chaincode:
-    system:
-        tlcc: enable
+TLCC is integrated into a fabric peer as system chaincode and thus must be build together with the peer binary.
```

Comment:
`build` -> `built` 

alternatively, "must be built into the peer binary. This means the peer must be compiled and the resulting binaries replace those from a standard fabric distribution."

Feel free to keep as much or as little of this as you want. I can make the typo change barring objections.